cassandra_test.go

```diff
@@ -1977,6 +1977,40 @@ func TestNegativeStream(t *testing.T) {
 	}
 }
 
+func TestProtocolV2StreamExhaustion(t *testing.T) {
```

Comment:
I'm not too familiar with the testing functions - which unit test functions should I use?

I wanted a roundtrip test for my use case because I wasn't sure that stream exhaustion was the only problem.